packages/react-components/react-radio/library/src/components/Radio/useRadioStyles.styles.ts

```diff
@@ -3,6 +3,7 @@ import { tokens } from '@fluentui/react-theme';
 import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
 import type { RadioSlots, RadioState } from './Radio.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import * as semanticTokens from '@fluentui/semantic-tokens';
```

Comment:
It's better for tree-shaking to destructure this import. E.g.:

`import { _ctrlRadioChoiceBaseSize, ... } from '@fluentui/semantic-tokens';`

See: https://web.dev/articles/reduce-javascript-payloads-with-tree-shaking#what_is_tree_shaking

Reply:
Confirmed with Shift but this import format is fully tree shakable (thanks to flat export from library package) - we also import and re-export the required legacy tokens as a flat export to solve them not being tree-shakeable in default ESBuild